dns-controller/cmd/dns-controller/main.go

```diff
@@ -65,6 +65,8 @@ func main() {
 	flags.StringVar(&gossipListen, "gossip-listen", "0.0.0.0:3998", "The address on which to listen if gossip is enabled")
 	flags.StringVar(&gossipSecret, "gossip-secret", gossipSecret, "Secret to use to secure gossip")
 	flags.StringVar(&watchNamespace, "watch-namespace", "", "Limits the functionality for pods, services and ingress to specific namespace, by default all")
+	flag.IntVar(&route53.MaxBatchSize, "route53-batch-size", 900, "Maximum number of operations performed per changeset batch")
```

Comment:
After setting the default value (see below), this can actually be:

`flag.IntVar(&route53.MaxBatchSize, "route53-batch-size", route53.MaxBatchSize, "Maximum number of operations performed per changeset batch")`

This avoids duplicating the 900 default value